modules/contribute/pages/set-up-repository.adoc

```diff
@@ -0,0 +1,75 @@
+= Set Up Your Local Repository
+
+You only need to do these steps the first time you contribute to a repository.
+
+== Fork the Repository
+
+Before you can contribute changes to the Stargate documentation, you must fork
+the appropriate repository to your personal GitHub account. 
+Then you can clone that personal repository to your current working computer.
+
+Go to the appropriate repository on GitHub and click btn:[Fork].
+See xref:repositories.adoc#repo-urls[Repository Organization] for the list of
+active repositories.
+
+.How to quickly find the right repository
+****
+To quickly find a page's source file, click on the image:edit.svg[,16,role=icon]
+edit icon at the top of the page.
+You'll be taken to the page's repository.
+Click btn:[Fork].
+
+If there isn't an image:edit.svg[,16,role=icon] edit icon on the page, then the
+repository can't accept changes using the regular workflow.
+****
+
+== Clone the Repository
+
+After you fork the repository, clone it to your device.
+
+. Create or select the folder where you want to store the repository on your device.
+. Open a terminal and change into the selected folder.
+
+ $ cd path/to/folder
+
+. Go to your repository on GitHub, and click btn:[Clone or download].
+. Click the Clipboard icon to copy the repository's Git URL.
+. In your terminal type `git clone`, paste the copied URL, and press kbd:[Enter].
+For example, to clone your fork of the `docs` repository, you'd enter:
+
+ $ git clone git@github.com:this-is-you/docs.git
+
+. Change into the repository folder created by the clone operation.
+
+ $ cd stargate/docs
+
+== Sync with the Upstream Repository
+
+You need to connect your local repository to the upstream repository so the two repositories can stay in sync.
+
+. Go to the upstream repository on GitHub, and click btn:[Clone or download].
+. Click the Clipboard icon to copy the repository's Git URL.
+. In your terminal type `git remote add upstream`, paste the copied URL, and press kbd:[Enter].
+For example, to connect to the remote upstream stargate/docs repository, you'd enter:
+
+ $ git remote add upstream git@github.com:stargate/docs.git
+
+. View the configured upstream and origin values and confirm the URLs are correct.
++
+--
+ $ git remote -v
```

Comment:
Provide an example result that could be returned.

Reply:
added a return